sync/sync.go

```diff
@@ -41,13 +44,17 @@ type Reader interface {
 	StartingBlockNumber() (uint64, error)
 	HighestBlockHeader() *core.Header
 	SubscribeNewHeads() HeaderSubscription
+
+	Pending() (*Pending, error)
+	PendingBlock() *core.Block
+	PendingState() (core.StateReader, func() error, error)
```

Comment:
PendingNewClasses() as well?

Reply:
I am not sure what you mean. I just moved the function from Blockchain Reader which referred to Pending.